From 49d664c18719f6701fe4ce9e1e72782e0975e718 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 31 Aug 2017 00:46:03 -0700
Subject: [PATCH 4/4] Avoid stripping the -g option

This configure macro strips the -g option from CFLAGS if the
--enable-debug option is not passed. But --enable-debug enables more of
a curl developer debug build (AFAICS), less intended for a distro build
that simply wants to make sure debuginfo is produced and later stripped
and distributed in the separate debuginfo package.

Disable that macro from being called to avoid the -g option stripping.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: Arzhan Kinzhalin <arzhan.i.kinzhalin@intel.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 1b7cfc3..18f29b9 100755
--- a/configure.ac
+++ b/configure.ac
@@ -260,7 +260,6 @@ dnl **********************************************************************
 
 CURL_CHECK_COMPILER
 CURL_SET_COMPILER_BASIC_OPTS
-CURL_SET_COMPILER_DEBUG_OPTS
 CURL_SET_COMPILER_OPTIMIZE_OPTS
 CURL_SET_COMPILER_WARNING_OPTS
 
-- 
2.15.0

